changelog/v1.15.0-beta15/add-direct-route-local-gateway-proxy.yaml

```
@@ -0,0 +1,6 @@
+changelog:
+- type: NEW_FEATURE
```

Our changelogs are user facing and the description will show up in the changelog if it is a type that is a user facing change.
Therefore I would suggest the following:
```suggestion
- type: NON_USER_FACING
```